### DISCUSSION TITLE
Odd question, but why is the app still in version 0?

### DISCUSSION THREAD

**Discussion by AlmightyWings232** [TeamNewPipe/NewPipe#10203] [Questions] [answered]
I understand it's an odd question but why is the app still in version 0.y.z. Does that mean it's still in initial development? At least, Semantic Versioning says it's meant for initial development (https://semver.org/#spec-item-4)

**Comment by SameenAhnaf** [comment, marked as answer]
> Does that mean it's still in initial development?

Yes, this is already mentioned in repo description. For more details, click [here](https://github.com/TeamNewPipe/NewPipe).

*WARNING: THIS APP IS IN BETA, SO YOU MAY ENCOUNTER BUGS. IF YOU DO, OPEN AN ISSUE IN OUR GITHUB REPOSITORY BY FILLING OUT THE ISSUE TEMPLATE.*

**Reply by AlmightyWings232** [reply]
Aaah, right. Been using NewPipe for the longest time, didn't even realise it's in beta. Thank you!
